web/src/params/id.ts

```diff
@@ -0,0 +1,5 @@
+import type { ParamMatcher } from '@sveltejs/kit';
+
+export const match: ParamMatcher = (param: string) => {
+  return /^[\dA-Fa-f]{8}(?:\b-[\dA-Fa-f]{4}){3}\b-[\dA-Fa-f]{12}$/.test(param);
```

Comment:
Ah comment of what that does may be helpful here. It avoids having to take the extra 10 seconds to figure it out.

Reply:
Thank you :)